### PR TITLE
[Learn] added quiz in learning paths

### PR DESCRIPTION
**Pull request by NikhilSharma03**
Signed-off-by: Nikhil Sharma <nikhilsharmamusic2000@gmail.com>

**Description**
- added quiz in learning paths
### Flow

- User finishes the course and will have the option to take the Quiz by using the `Take Quiz`  button. 
- User will be redirected to the login page for user authentication(since it is not connected with any backend, so just using test login data for demo)
- User takes the quiz and if he passes he will get the `Download Certificate`  button.

<br />

https://user-images.githubusercontent.com/58226527/151354648-b3f7093e-65ba-43ac-a05c-f15f4acc7919.mp4

**Notes for Reviewers**
**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 1f710a3e8d834be77b307b4c598eff689e8aadef at: https://61f28baed7e9aab8931c8afb--layer5ng.netlify.app

**Comment by NikhilSharma03** [comment]
@warunicorn19 I'll remove the `take quiz` button and then let's get this merged so everyone can contribute to it. Does that sound good or we should take another route?

**Comment by warunicorn19** [comment]
Yup sounds good

**Comment by NikhilSharma03** [comment]
Done. Also Sorry, i didn't quite understand your meaning while the meet :sweat_smile:  . After recollecting, i understand Oh! this was what we were talking about.

File: src/components/Learn-Components/Pagination/index.js [hunk 1]
```diff
@@ -50,13 +50,8 @@ const Pagination = ({ TOCData, chapterData, location, showQuizModal }) => {
           </div>
         </div>
       </PaginationWrapper>
-    ) : (
```
Comment (warunicorn19) [inline src/components/Learn-Components/Pagination/index.js]: @NikhilSharma03 could you please just comment it out, instead of removing it.

Comment (warunicorn19) [reply src/components/Learn-Components/Pagination/index.js]: Cause I see that the URL is not changing so it's gonna be hard to locate if we remove the button

Comment (NikhilSharma03) [reply src/components/Learn-Components/Pagination/index.js]: Yup, that would be better. Done.

**Comment by l5io** [comment]
🚀 Preview for commit 961013806fa832452ada1d0b79c121dfb019bd81 at: https://61f8e71f521d215d0601fc49--layer5ng.netlify.app

**Review by warunicorn19 — APPROVED** [review]
Looking good, thank you @NikhilSharma03

**Pull request merged**